src/content/en/fundamentals/instant-and-offline/get-started/textcontent-3.md

```diff
@@ -43,25 +43,15 @@ or functionality. It doesn't sound that useful, but it is. For example, this lit
 
 ```
 function sortables_init() {
-
     // Find all tables with class sortable and make them sortable
-
     if (!document.getElementsByTagName) return;
-
     tbls = document.getElementsByTagName("table");
```

Comment:
Please replace this line with `let tbls = document.getElementsByTagName("table");` or `var tbls = document.getElementsByTagName("table");`. Without a variable instantiation here, after calling `shortables_init()` the `tbls` would leak out to the global scope. Super minor, but it helps for our inline code samples to catch these things :)